crates/swc_ecma_transforms_compat/src/macros.rs

```diff
@@ -168,3 +168,159 @@ macro_rules! impl_fold_fn {
         }
     };
 }
+
+macro_rules! impl_visit_mut_fn {
```

Comment:
There are few places still using `impl_fold_fn` macro, so PR do not delete it yet.